scripts/setup-pvs-proxy-rules

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]]
+then
+    echo "Usage: $0 [add|remove] <private_xs_path>"
```

Comment:
`echo "Usage: $0 [add|remove] <private_xs_path>" >&2` directs this to stderr. This avoids that an process consuming the output would read the error message.